.github/workflows/azure-dev.yml

```diff
@@ -24,7 +24,6 @@ jobs:
       AZURE_TENANT_ID: ${{ vars.AZURE_TENANT_ID }}
```

Comment:
Please look at the azure.yaml file to remove the post-provision hooks to see if the app still works.  

The goal should be to be able to run azd provision and then F5 in Visual Studio.

Reply:
This repo doesn't contain hooks in azure.yaml file, maybe that's what you want to say here? https://github.com/jongio/chat-with-your-data-solution-accelerator/pull/2/files#diff-1ba5bd52036276068bc787a1be3dfa49aa920317ab02fa460c99246a78e588f3R11-R17

Reply:
I commented on the wrong project you can disregard.